modal/sandbox.py

```diff
@@ -234,6 +235,8 @@ async def _load(self: _Sandbox, resolver: Resolver, _existing_object_id: Optiona
                 name=name,
                 experimental_options=experimental_options,
             )
+            if idle_timeout is not None:
```

Comment:
J/w but is there a reason you're not passing this directly to `api_pb2.Sandbox`?